src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -2362,7 +2362,7 @@ public override BoundNode VisitConditionalAccess(BoundConditionalAccess node)
 
         public override BoundNode VisitConditionalOperator(BoundConditionalOperator node)
```

Comment:
>VisitConditionalOperator [](start = 34, length = 24)

I suspect this whole method would be simpler if split completely for the `ref` case and the value case.  It seems like to cost in clarity of trying to do both at once is high.